libs/message-coder/package.json

```diff
@@ -0,0 +1,47 @@
+{
+  "name": "@votingworks/message-coder",
```

Comment:
The other stuff @adghayes was working on could benefit from this, as could any other future direct-to-USB drivers we write in the future. One potential place that could use this right now is `ballot-encoder` where it'd replace `BitReader` and `BitWriter`.

Reply:
If I started working on the `print-scan` work again, I don't think I would want to use this library.

The data structures I was encoding there were usually commands or simple numbers / enumerations. Using simple helpers worked fine / seems preferable to me. For example:

```js
  async setPrintingSpeed(
    printingSpeed: PrintingSpeed
  ): Promise<USBOutTransferResult> {
    return this.transferOutGeneric([
      ...SET_PRINTING_SPEED,
      PRINTING_SPEED_CODES[printingSpeed],
    ]);
  }
```

I could be convinced that the additional typing and defining structures is helpful for reading the code or being safer, but I don't expect to iterate on the drivers like other parts of the code base. So if it works, that's good enough for me. I'm less concerned about strictly and clearly defining the form of each command in code - I'll let the driver command manual do that.

On the other hand, I did copy over a good bit of code from `ballot-encoder` and there's a lot of that which could be shared.

Reply:
On second thought, I think there are a couple commands with hefty content where this could be helpful. So I could see using it for just those heavier endpoints.

Reply:
E.g. https://github.com/votingworks/vxsuite-print-scan/blob/main/services/print-scan/src/paper_handler/driver/scanner_capability.ts#L5